### PR TITLE
Update the basic tutorial and Amsterdam Tutorial after the new version release.

### PR DESCRIPTION
**Pull request by oguzhan**
* Updates the notebook(s) after major pyGUS release v1.0
* Adds Amsterdam Trees Data
* Adds Notebook that gets Amsterdam Trees Data as input and runs GUS on it.
* Uses utility functions to map `lat, lng` variables to x,y pos on the Mesa grid.

### CODE REVIEW

File: notebooks/Amsterdam.ipynb [hunk 1]
```diff
@@ -0,0 +1,1848 @@
...
+    "#### 1.4 Map LAT,LNG to x,y POS on MESA Grid."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_pop = df_pop.apply(latlng_to_xy, axis=1)"
```
Comment (lucid-jake) [inline notebooks/Amsterdam.ipynb:552]: @oguzhan I think this method might be wrong. The maps you generated of A'dam seem to me to be on their side...
The Latitude is the north/south measure, and Longitude, the east/west measure. So I think line 55 here should be inverted, right?
image

Comment (oguzhan) [reply notebooks/Amsterdam.ipynb:552]: you think the method from utm returns the results inverted or we assigned it to the wrong vars after the calculation?

Comment (lucid-jake) [reply notebooks/Amsterdam.ipynb:552]: I think longitude should be X, and latitude should be Y

Comment (lucid-jake) [reply notebooks/Amsterdam.ipynb:552]: Oh, never mind, the UTM method returns "easting" and "northing" in that order, which would be correct. I was just grasping for reasons why the plots looked so weird, but I guess we would need more trees to recognise the city

**Review by lucid-jake — APPROVED** [review]
Looks good to me :)

**Pull request merged**